### ISSUE TITLE
Accessibility: Cannot label input when using as a form control

### ISSUE THREAD

**Issue by dylan-cancelliere** [jackocnr/intl-tel-input#2040]
Hi, using intel-tel-input in a react project in a form. According to aria guidelines, form controls should be associated with a label, see https://www.w3.org/WAI/ARIA/apg/practices/names-and-descriptions/#naming_with_labels.

In my experience, this typically is done with a `<label>` element directly above the `<input>` being labeled. Alternatively, the `aria-labelledby` attribute can be used to specify an ID.

My proposals:

1. If possible, add a prop that allows specifying a `<label>` that will be directly above the `<input>` in the tree.
2. More simply, and maybe(?) more appropriate in this case, add a prop that passes through a `aria-labelledby` id to the `<input>` component so that the user can specify their own label.

Thanks for your time, and I should clarify I am by no means an accessibility expert, just working on making a site more accessible.

**Comment by jackocnr** [comment]
Having a label directly before the input element would cause big problems for the country dropdown layout/styling. But your option 2 should be achievable already using the `inputProps` prop in the react component?

**Comment by dylan-cancelliere** [comment]
You're right, that works perfectly. I missed `inputProps` in the docs, thanks for the quick reply
